### PR TITLE
Add introduction of Alpha feature EvenPodsSpread to 1.16

### PR DESCRIPTION
**Pull request by saschagrunert**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Adds the missing release notes about EvenPodsSpread to the 1.16 changelog.

**Which issue(s) this PR fixes**:

Relates to https://github.com/kubernetes/sig-release/pull/801

**Special notes for your reviewer**:

/cc @Huang-Wei 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.16.md [hunk 1]
```diff
@@ -339,6 +339,7 @@ The main themes of this release are:
...
+- EvenPodsSpread feature has been added in alpha. This feature introduced an API field `pod.spec.topologySpreadConstraints` to enable users control how Pods are spread across the cluster among failure-domains. ([#77327](https://github.com/kubernetes/kubernetes/pull/77327), [#77760](https://github.com/kubernetes/kubernetes/pull/77760), [#77828](https://github.com/kubernetes/kubernetes/pull/77828), [#79062](https://github.com/kubernetes/kubernetes/pull/79062), [#80011](https://github.com/kubernetes/kubernetes/pull/80011), [#81068](https://github.com/kubernetes/kubernetes/pull/81068), [@Huang-Wei](https://github.com/Huang-Wei))
```
Comment (sftim) [inline CHANGELOG-1.16.md]: My suggestion for wording:
```markdown
- Pod spread constraints have been added in alpha. You can use these constraints to control how Pods are spread across the cluster among failure-domains. ([#77327](https://github.com/kubernetes/kubernetes/pull/77327), [#77760](https://github.com/kubernetes/kubernetes/pull/77760), [#77828](https://github.com/kubernetes/kubernetes/pull/77828), [#79062](https://github.com/kubernetes/kubernetes/pull/79062), [#80011](https://github.com/kubernetes/kubernetes/pull/80011), [#81068](https://github.com/kubernetes/kubernetes/pull/81068), [@Huang-Wei](https://github.com/Huang-Wei))
```
(also made in https://github.com/kubernetes/website/pull/16446)

Comment (saschagrunert) [reply CHANGELOG-1.16.md]: Changed as suggested :+1:

**Comment by Huang-Wei** [comment]
/close
in favor of #82853

**Pull request closed without merging**